src/pages/devResources/devResources.js

```diff
@@ -166,9 +166,9 @@ const mapStateToProps = state => ({
 
 const ConnectedDevResourcesPage = connect(
   mapStateToProps,
-  mapDispatchToProps
+  mapDispatchToProps,
```

Comment:
I would not include this file in the PR, not sure what it's adding the changes should be limited to just about and masthead. I'm guessing both of these changes would probably be flagged by lint.